README.md

```diff
@@ -60,6 +63,17 @@ conda install -c codecarbon -c conda-forge codecarbon
 `codecarbon` is now installed in your the local environment
 
 
+# Infrastructure Support
+Currently the package supports following hardware infrastructure.
+
+### GPU
+- Tracks Nvidia GPUs power consumption using pynvml library.
```

Comment:
Might be worth adding that no additional installation is required from the user, as pynvml is tracked as a package dependency.